src/scopes/activityReport/index.test.js

```diff
@@ -967,6 +975,164 @@ describe('filtersToScopes', () => {
+      });
+
+      const dummyProgram = {
+        startYear: '2020',
+        startDate: '2020-09-01',
+        endDate: '2020-09-02',
+        status: 'Active',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      };
+
+      programOne = await Program.create({
+        ...dummyProgram,
+        id: grantIncluded1.id,
+        name: faker.name.findName(),
+        grantId: grantIncluded1.id,
+        programType: 'EHS',
+      });
+      programTwo = await Program.create({
+        ...dummyProgram,
+        id: grantIncluded2.id,
+        name: faker.name.findName(),
+        grantId: grantIncluded2.id,
+        programType: 'EHS',
+      });
+      programThree = await Program.create({
+        ...dummyProgram,
+        id: grantExcluded.id,
+        name: faker.name.findName(),
+        grantId: grantExcluded.id,
+        programType: 'HS',
+      });
+
+      reportOne = await ActivityReport.create({ ...draftReport });
+      reportTwo = await ActivityReport.create({ ...draftReport });
+      reportThree = await ActivityReport.create({ ...draftReport });
+
+      activityRecipientIncluded1 = await ActivityRecipient.create({
+        activityReportId: reportOne.id,
+        grantId: grantIncluded1.id,
+      });
+      activityRecipientIncluded2 = await ActivityRecipient.create({
+        activityReportId: reportTwo.id,
+        grantId: grantIncluded2.id,
+      });
+      activityRecipientExcluded = await ActivityRecipient.create({
+        activityReportId: reportThree.id,
+        grantId: grantExcluded.id,
+      });
+
+      possibleIds = [
+        reportOne.id,
+        reportTwo.id,
+        reportThree.id,
+        globallyExcludedReport.id,
+      ];
+    });
+
+    afterAll(async () => {
+      await ActivityRecipient.destroy({
+        where: {
+          id: [
+            activityRecipientIncluded1.id,
+            activityRecipientIncluded2.id,
+            activityRecipientExcluded.id,
+          ],
+        },
+      });
+      await ActivityReport.destroy({
+        where: { id: [reportOne.id, reportTwo.id, reportThree.id] },
+      });
+
+      await Program.destroy({
+        where: { id: [programOne.id, programTwo.id, programThree.id] },
+      });
+
+      await Grant.destroy({
+        where: { id: [grantIncluded1.id, grantIncluded2.id, grantExcluded.id] },
+      });
+      await Grantee.destroy({
+        where: { id: [granteeIncluded1.id, granteeIncluded2.id, granteeExcluded.id] },
+      });
+    });
+
+    it('includes program type', async () => {
+      const filters = { 'programType.in': ['EHS', 'HS'] };
+      const scope = filtersToScopes(filters);
+      const found = await ActivityReport.findAll({
+        where: { [Op.and]: [scope, { id: possibleIds }] },
+      });
+      expect(found.length).toBe(3);
+      expect(found.map((f) => f.id))
+        .toEqual(expect.arrayContaining([reportOne.id, reportTwo.id, reportThree.id]));
+    });
+
+    it('excludes program type', async () => {
```

Comment:
Can we add a test case that checks program type not being multiple values ie `programType.nin': ['EHS', 'HS']`?